tests/syscall/fs_linux/host/host.c

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#if defined(_WIN32)
+#include <windows.h>
+#endif
+#include <openenclave/host.h>
+#include <openenclave/internal/syscall/host.h>
+#include <openenclave/internal/tests.h>
+#include <stdio.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include "fs_u.h"
+
+#define SKIP_RETURN_CODE 2
+
+int rmdir(const char* path);
+
+int test_fs_posix(
+    const char* enclave_path,
+    const char* src_dir,
+    const char* tmp_dir)
+{
+    const uint32_t flags = oe_get_create_flags();
+    if ((flags & OE_ENCLAVE_FLAG_SIMULATE))
+    {
+        printf("=== Skipped unsupported test in simulation mode (sealKey)\n");
+        return SKIP_RETURN_CODE;
+    }
+
+    rmdir(tmp_dir);
+
+    const oe_enclave_type_t type = OE_ENCLAVE_TYPE_SGX;
+    oe_enclave_t* enclave = NULL;
+    oe_result_t r =
+        oe_create_fs_enclave(enclave_path, type, flags, NULL, 0, &enclave);
+    OE_TEST(r == OE_OK);
+
+    r = test_fs(enclave, src_dir, tmp_dir);
+    OE_TEST(r == OE_OK);
+
+    r = oe_terminate_enclave(enclave);
+    OE_TEST(r == OE_OK);
+
+    printf("=== passed all tests (hostfs)\n");
+
+    return 0;
+}
+
+#if defined(_WIN32)
```

Comment:
I don't follow why there is _WIN32 section in a file under fs_linux.   What does fs_linux mean?
(Answering Simon's question would likely answer mine as well.)

Reply:
Fixed with Simon's suggestion, thanks.